### PR TITLE
Improve ColorNegate doc

### PR DESCRIPTION
**Pull request by rocky**
Had thought I had included this previously, but with various merges and rebases it got lost.

### CODE REVIEW

File: mathics/builtin/colors/color_operations.py [hunk 1]
```diff
@@ -201,26 +203,32 @@ def eval(self, input, colorspace, evaluation):
...
+    :WMA:
+    https://reference.wolfram.com/language/ref/ColorNegate.html</url>)
 
     <dl>
-      <dt>'ColorNegate[$image$]'
-      <dd>returns the negative of $image$ in which colors have been negated.
-
       <dt>'ColorNegate[$color$]'
-      <dd>returns the negative of a color.
+      <dd>returns the RGB color when it is subtracted from white.
```
Comment (TiagoCavalcante) [inline mathics/builtin/colors/color_operations.py:212]: Maybe `returns the negative of a color, that is, the RGB color subtract from white`?

File: mathics/builtin/colors/color_operations.py [hunk 2]
```diff
@@ -201,26 +203,32 @@ def eval(self, input, colorspace, evaluation):
...
 
-      Yellow is RGBColor[1.0, 1.0, 0.0]
-      >> ColorNegate[Yellow]
-       = RGBColor[0., 0., 1.]
+      <dt>'ColorNegate[$image$]'
+      <dd>returns an image where each pixel in the image is replaced \
```
Comment (TiagoCavalcante) [inline mathics/builtin/colors/color_operations.py:215]: Maybe `returns an image where each pixel has its color negated`?

**Review by TiagoCavalcante — APPROVED** [review]
LGTM

File: mathics/builtin/colors/color_operations.py (hunk 2 above)
Comment (rocky) [reply mathics/builtin/colors/color_operations.py:215]: Yes, that is better.

File: mathics/builtin/colors/color_operations.py (hunk 1 above)
Comment (rocky) [reply mathics/builtin/colors/color_operations.py]: subtract -> subtracted, but yes this too is clearer.

**Pull request merged**